### ISSUE TITLE
1.0.14 Crash after selecting multiple songs either with/without playing them crashes program

### ISSUE THREAD

**Issue by pkbott** [mpogue2/SquareDesk#1239]
I tried just opening about 14 or so songs without playing them and crashed the program.  After doing that multiple times, I then tried doing the same with playing the songs for about 3-5 seconds each and was able to crash the program that way also.

Time Awake Since Boot: 1600 seconds

System Integrity Protection: enabled

Crashed Thread:        0  CrBrowserMain  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_PROTECTION_FAILURE at 0x000000016ab93f50
Exception Codes:       0x0000000000000002, 0x000000016ab93f50

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [1269]

VM Region Info: 0x16ab93f50 is in 0x167390000-0x16ab94000;  bytes after start: 58736464  bytes before end: 175
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      mapped file                 15ad18000-15ae18000    [ 1024K] r--/rwx SM=COW  Object_id=ef50c10
      GAP OF 0xc578000 BYTES
--->  STACK GUARD                 167390000-16ab94000    [ 56.0M] ---/rwx SM=NUL  stack guard for thread 0
      Stack                       16ab94000-16b390000    [ 8176K] rw-/rwx SM=SHM  thread 0

**Comment by pkbott** [comment]
If while one song was playing I selected another song and then playing it I could not replicated the bug.  Hoverver, if I started a song then stopped it then chose another song started it stopped it and continued the process I was able to crash the program.  If I didn't stop the first song and just selected the next song, no crash.

**Comment by pkbott** [comment]
I tried a combination of the two methods and crashed the program as well.

**Comment by mpogue2** [comment]
I tried for about 1/2 hour, and I can't get it to fail! Of course it won't fail for me! :-) :-)

**Comment by pkbott** [comment]
Of course. When I did it I only let the songs play for less than 30 seconds
each before moving to the next song if that helps.

**Comment by mpogue2** [comment]
I tried that...I'll keep trying, because I need to reproduce it here to be able to fix it.  And thanks again for the report! :-)

**Comment by mpogue2** [comment]
Closing as not reproducible with current builds.
